src/main/kotlin/ru/ozon/ideplugin/kelp/KelpConfig.kt

```diff
@@ -67,12 +67,12 @@ class KelpConfig(
 
     @Serializable
     class DemoApp(
-        val functionFqnPrefix: String,
-        val functionSimpleNamePrefix: String? = null,
-        val appPackageName: String,
-        val componentDeeplink: String,
-        val intentionName: String = "🚀 Open in design system demo app",
-        val apkInstalling: ApkInstalling? = null,
+            val functionFqnPrefix: String,
+            val functionSimpleNamePrefix: String? = null,
+            val appPackageName: String,
+            val componentDeeplink: String,
+            val intentionName: String = KelpBundle.message("openInDemoAppIntentionName", "\uD83D\uDE80"),
```

Comment:
Is it needed to pass an emoji as a parameter? Maybe bake it into the string in `.properties` file?

Reply:
There is problem with encoding, not working like that